apps/vr-tests-web-components/src/stories/button/button.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28797&commitId=67c4a031aee9b97698ad76945f4780379278758f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button</b> 144 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Circular.default.chromium.png|0| Added|
|Button.Circular.hover.chromium.png|0| Added|
|Button.Circular.pressed.chromium.png|0| Added|
|Button.Default.default.chromium.png|0| Added|
|Button.Default.hover.chromium.png|0| Added|
|Button.Default.pressed.chromium.png|0| Added|
|Button.Large Icon Only - Dark Mode.default.chromium.png|0| Added|
|Button.Large Icon Only - Dark Mode.hover.chromium.png|0| Added|
|Button.Large Icon Only - Dark Mode.pressed.chromium.png|0| Added|
|Button.Large Icon Only.default.chromium.png|0| Added|
|Button.Large Icon Only.hover.chromium.png|0| Added|
|Button.Large Icon Only.pressed.chromium.png|0| Added|
|Button.Large With Icon - Dark Mode.default.chromium.png|0| Added|
|Button.Large With Icon - Dark Mode.hover.chromium.png|0| Added|
|Button.Large With Icon - Dark Mode.pressed.chromium.png|0| Added|
|Button.Large With Icon - RTL.default.chromium.png|0| Added|
|Button.Large With Icon - RTL.hover.chromium.png|0| Added|
|Button.Large With Icon - RTL.pressed.chromium.png|0| Added|
|Button.Large With Icon.default.chromium.png|0| Added|
|Button.Large With Icon.hover.chromium.png|0| Added|
|Button.Large With Icon.pressed.chromium.png|0| Added|
|Button.Large.default.chromium.png|0| Added|
|Button.Large.hover.chromium.png|0| Added|
|Button.Large.pressed.chromium.png|0| Added|
|Button.Medium Icon Only - Dark Mode.default.chromium.png|0| Added|
|Button.Medium Icon Only - Dark Mode.hover.chromium.png|0| Added|
|Button.Medium Icon Only - Dark Mode.pressed.chromium.png|0| Added|
|Button.Medium Icon Only.default.chromium.png|0| Added|
|Button.Medium Icon Only.hover.chromium.png|0| Added|
|Button.Medium Icon Only.pressed.chromium.png|0| Added|
|Button.Medium With Icon - Dark Mode.default.chromium.png|0| Added|
|Button.Medium With Icon - Dark Mode.hover.chromium.png|0| Added|
|Button.Medium With Icon - Dark Mode.pressed.chromium.png|0| Added|
|Button.Medium With Icon - RTL.default.chromium.png|0| Added|
|Button.Medium With Icon - RTL.hover.chromium.png|0| Added|
|Button.Medium With Icon - RTL.pressed.chromium.png|0| Added|
|Button.Medium With Icon.default.chromium.png|0| Added|
|Button.Medium With Icon.hover.chromium.png|0| Added|
|Button.Medium With Icon.pressed.chromium.png|0| Added|
|Button.Medium.default.chromium.png|0| Added|
|Button.Medium.hover.chromium.png|0| Added|
|Button.Medium.pressed.chromium.png|0| Added|
|Button.Outline - Dark Mode.default.chromium.png|0| Added|
|Button.Outline - Dark Mode.hover.chromium.png|0| Added|
|Button.Outline - Dark Mode.pressed.chromium.png|0| Added|
|Button.Outline Disabled - Dark Mode.default.chromium.png|0| Added|
|Button.Outline Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button.Outline Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button.Outline Disabled Focusable - Dark Mode.default.chromium.png|0| Added|
|Button.Outline Disabled Focusable - Dark Mode.hover.chromium.png|0| Added|
|Button.Outline Disabled Focusable - Dark Mode.pressed.chromium.png|0| Added|
|Button.Outline Disabled Focusable.default.chromium.png|0| Added|
|Button.Outline Disabled Focusable.hover.chromium.png|0| Added|
|Button.Outline Disabled Focusable.pressed.chromium.png|0| Added|
|Button.Outline Disabled.default.chromium.png|0| Added|
|Button.Outline Disabled.hover.chromium.png|0| Added|
|Button.Outline Disabled.pressed.chromium.png|0| Added|
|Button.Outline.default.chromium.png|0| Added|
|Button.Outline.hover.chromium.png|0| Added|
|Button.Outline.pressed.chromium.png|0| Added|
|Button.Primary - Dark Mode.default.chromium.png|0| Added|
|Button.Primary - Dark Mode.hover.chromium.png|0| Added|
|Button.Primary - Dark Mode.pressed.chromium.png|0| Added|
|Button.Primary Disabled - Dark Mode.default.chromium.png|0| Added|
|Button.Primary Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button.Primary Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button.Primary Disabled Focusable - Dark Mode.default.chromium.png|0| Added|
|Button.Primary Disabled Focusable - Dark Mode.hover.chromium.png|0| Added|
|Button.Primary Disabled Focusable - Dark Mode.pressed.chromium.png|0| Added|
|Button.Primary Disabled Focusable.default.chromium.png|0| Added|
|Button.Primary Disabled Focusable.hover.chromium.png|0| Added|
|Button.Primary Disabled Focusable.pressed.chromium.png|0| Added|
|Button.Primary Disabled.default.chromium.png|0| Added|
|Button.Primary Disabled.hover.chromium.png|0| Added|
|Button.Primary Disabled.pressed.chromium.png|0| Added|
|Button.Primary.default.chromium.png|0| Added|
|Button.Primary.hover.chromium.png|0| Added|
|Button.Primary.pressed.chromium.png|0| Added|
|Button.Rounded.default.chromium.png|0| Added|
|Button.Rounded.hover.chromium.png|0| Added|
|Button.Rounded.pressed.chromium.png|0| Added|
|Button.Small Icon Only - Dark Mode.default.chromium.png|0| Added|
|Button.Small Icon Only - Dark Mode.hover.chromium.png|0| Added|
|Button.Small Icon Only - Dark Mode.pressed.chromium.png|0| Added|
|Button.Small Icon Only.default.chromium.png|0| Added|
|Button.Small Icon Only.hover.chromium.png|0| Added|
|Button.Small Icon Only.pressed.chromium.png|0| Added|
|Button.Small With Icon - Dark Mode.default.chromium.png|0| Added|
|Button.Small With Icon - Dark Mode.hover.chromium.png|0| Added|
|Button.Small With Icon - Dark Mode.pressed.chromium.png|0| Added|
|Button.Small With Icon - RTL.default.chromium.png|0| Added|
|Button.Small With Icon - RTL.hover.chromium.png|0| Added|
|Button.Small With Icon - RTL.pressed.chromium.png|0| Added|
|Button.Small With Icon.default.chromium.png|0| Added|
|Button.Small With Icon.hover.chromium.png|0| Added|
|Button.Small With Icon.pressed.chromium.png|0| Added|
|Button.Small.default.chromium.png|0| Added|
|Button.Small.hover.chromium.png|0| Added|
|Button.Small.pressed.chromium.png|0| Added|
|Button.Square.default.chromium.png|0| Added|
|Button.Square.hover.chromium.png|0| Added|
|Button.Square.pressed.chromium.png|0| Added|
|Button.Subtle - Dark Mode.default.chromium.png|0| Added|
|Button.Subtle - Dark Mode.hover.chromium.png|0| Added|
|Button.Subtle - Dark Mode.pressed.chromium.png|0| Added|
|Button.Subtle Disabled - Dark Mode.default.chromium.png|0| Added|
|Button.Subtle Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button.Subtle Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button.Subtle Disabled Focusable - Dark Mode.default.chromium.png|0| Added|
|Button.Subtle Disabled Focusable - Dark Mode.hover.chromium.png|0| Added|
|Button.Subtle Disabled Focusable - Dark Mode.pressed.chromium.png|0| Added|
|Button.Subtle Disabled Focusable.default.chromium.png|0| Added|
|Button.Subtle Disabled Focusable.hover.chromium.png|0| Added|
|Button.Subtle Disabled Focusable.pressed.chromium.png|0| Added|
|Button.Subtle Disabled.default.chromium.png|0| Added|
|Button.Subtle Disabled.hover.chromium.png|0| Added|
|Button.Subtle Disabled.pressed.chromium.png|0| Added|
|Button.Subtle.default.chromium.png|0| Added|
|Button.Subtle.hover.chromium.png|0| Added|
|Button.Subtle.pressed.chromium.png|0| Added|
|Button.Transparent - Dark Mode.default.chromium.png|0| Added|
|Button.Transparent - Dark Mode.hover.chromium.png|0| Added|
|Button.Transparent - Dark Mode.pressed.chromium.png|0| Added|
|Button.Transparent Disabled - Dark Mode.default.chromium.png|0| Added|
|Button.Transparent Disabled - Dark Mode.hover.chromium.png|0| Added|
|Button.Transparent Disabled - Dark Mode.pressed.chromium.png|0| Added|
|Button.Transparent Disabled Focusable - Dark Mode.default.chromium.png|0| Added|
|Button.Transparent Disabled Focusable - Dark Mode.hover.chromium.png|0| Added|
|Button.Transparent Disabled Focusable - Dark Mode.pressed.chromium.png|0| Added|
|Button.Transparent Disabled Focusable.default.chromium.png|0| Added|
|Button.Transparent Disabled Focusable.hover.chromium.png|0| Added|
|Button.Transparent Disabled Focusable.pressed.chromium.png|0| Added|
|Button.Transparent Disabled.default.chromium.png|0| Added|
|Button.Transparent Disabled.hover.chromium.png|0| Added|
|Button.Transparent Disabled.pressed.chromium.png|0| Added|
|Button.Transparent.default.chromium.png|0| Added|
|Button.Transparent.hover.chromium.png|0| Added|
|Button.Transparent.pressed.chromium.png|0| Added|
|Button.With Long Text - RTL.default.chromium.png|0| Added|
|Button.With Long Text - RTL.hover.chromium.png|0| Added|
|Button.With Long Text - RTL.pressed.chromium.png|0| Added|
|Button.With Long Text.default.chromium.png|0| Added|
|Button.With Long Text.hover.chromium.png|0| Added|
|Button.With Long Text.pressed.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>